src/modules/questions/questions.service.ts

```
@@ -510,6 +506,20 @@ export class QuestionsService {
             throw new BadRequestException('Matching questions must have options with matchWith specified.');
           }
           break;
+
+        case QuestionType.SUBJECTIVE:
+        case QuestionType.ESSAY:
+          // For subjective and essay questions, options are optional but if provided, they should be reference correct answers
+          const subjectiveCorrectOptions = options.filter(option => option.isCorrect);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)